### DISCUSSION TITLE
AVATTO Tuya WiFi IR Remote Control S08 Pro (with temperature/humidity sensor). WB3S->ESP8266 (ESP12-F)

### DISCUSSION THREAD

**Discussion by nikvostro** [arendst/Tasmota#12767] [Show and tell]
Sharing my experience of replacing WB3S with ESP8266 (ESP12-F).

I've bought this [device ](https://www.aliexpress.com/item/1005002505900746.html?aff_fcid=1c89e58b30fc4c48be3399635d554b08-1627394404997-01625-_pvOzfGS&aff_fsk=_pvOzfGS&af=%7B%22c%22%3A%22aehYn2niwHMQyElx1610218081%22%7D&aff_platform=api-new-link-generate&sk=_pvOzfGS&aff_trace_key=1c89e58b30fc4c48be3399635d554b08-1627394404997-01625-_pvOzfGS&terminal_id=7f09ad3b7b1a4cfca1a54678806efa41) and obtained WB3S chip instead of ESP8266.
![image](https://user-images.githubusercontent.com/83143316/127170886-94a78fbd-8999-45ea-a35e-3e1f0ee2195e.png)

Not a problem B-)

![image](https://user-images.githubusercontent.com/83143316/127171029-18efa925-69f3-4d47-8786-3d1a80eb5477.png)

![image](https://user-images.githubusercontent.com/83143316/127171056-101b636d-9869-49df-8ed1-4ba583e83d62.png)

There is a AHT10 temperature/humidity sensor is used (bottom left on pic). I've removed TuyaMCU chip and closed SCL/SDA pins on PCB directly to ESP8266. It works great with AHT1x Tasmota driver `#define USE_AHT1x` for my 2 Avatto devices:
![image](https://user-images.githubusercontent.com/83143316/127171110-53bbb892-27fa-4a39-98cd-53ea52f579f2.png)

![image](https://user-images.githubusercontent.com/83143316/127171146-6fcb72c3-ca13-4c42-8427-08a7d91877c6.png)

**Comment by hagge2000** [comment]
Great work! Could you provide me the firmware bin file?
